go.mod

```diff
@@ -6,56 +6,36 @@ require (
 	github.com/aws/aws-sdk-go v1.41.10
 	github.com/hashicorp/go-hclog v1.1.0
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/hashicorp/vault/api v1.4.1
+	github.com/hashicorp/vault/api v1.9.0
 	github.com/hashicorp/vault/sdk v0.4.1
```

Comment:
Any reason we aren't updating vault sdk as well?

Reply:
no reason, updated vault/sdk